testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -3327,6 +3327,25 @@ QUnit.module('datebox with time component', {
         assert.equal(dateBox.option('value').getMilliseconds(), 0, 'milliseconds has zero value');
     });
 
+    QUnit.test('invalid (by internal validation) value should be displayed if it is selected by time numberboxes (T939117)', function(assert) {
+        const $dateBox = $('#dateBox').dxDateBox({
+            type: 'datetime',
+            pickerType: 'calendar',
+            opened: true,
+            displayFormat: 'HH:mm',
```

Comment:
Is this prop necessary?

Reply:
it is not necessary, but it gives short understandable texts